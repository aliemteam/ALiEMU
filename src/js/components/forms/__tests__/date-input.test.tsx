import React from 'react';
import renderer from 'react-test-renderer';

import DateInput from '../date-input';

it('should render correctly', () => {
    const tree = renderer.create(
        <>
            <DateInput value="2018/11/12" />
            <DateInput value="201" delimiter="." />
        </>,
    );
    expect(tree).toMatchSnapshot();
});
