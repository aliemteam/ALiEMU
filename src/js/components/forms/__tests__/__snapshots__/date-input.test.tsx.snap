// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
Array [
  <div
    className="container"
  >
    <input
      className="input"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyPress={[Function]}
      pattern="\\\\d{4}/\\\\d{2}/\\\\d{2}"
      type="search"
      value="2018/11/12"
    />
  </div>,
  <div
    className="container"
  >
    <input
      className="input"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyPress={[Function]}
      pattern="\\\\d{4}/\\\\d{2}/\\\\d{2}"
      type="search"
      value="201"
    />
  </div>,
]
`;
